stream_alert/rule_processor/rules_engine.py

```diff
@@ -53,6 +53,7 @@ class StreamRules(object):
     def __init__(self, config):
         """Initialize a StreamRules instance to cache a StreamThreatIntel instance."""
         self._threat_intel = StreamThreatIntel.load_from_config(config)
+        self._required_outputs = set(resources.get_required_outputs())
```

Comment:
`get_required_outputs()` already returns a set, it doesn't need to be recreated as such. However, for clarity, you could rename `self._required_outputs` to `self._required_output_set` if you wanted

Reply:
good catch - the initial implementation of `get_required_outputs` actually returned a dict, which needed to be cast to `set`. updating now